manifests/0000_12_etcd-operator_01_config.crd.yaml

```diff
@@ -14,3 +14,10 @@ spec:
     - coreoperators
   subresources:
     status: {}
+  validation:
```

Comment:
Based on @sttts' suggestion here:
https://github.com/openshift/cluster-etcd-operator/pull/83#issuecomment-582351754

Reply:
From our slack discussion:

The CRD should not be in the ceo operator, but in openshift/api. Then you also get the schema generation for free (there is a makefile target for that). And you have to follow the approach kube-apiserver, i.e. using a yaml patch.

Reply:
cc @damemi – maybe you can help @retroflexer with moving the CRD manifest over.

Reply:
I will move the CRD manifest to openshift/api in a separate PR. I would let this PR go through without the CRD changes (I reverted the above change).